packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -151,7 +156,14 @@ export default function (
               return throwMsg === true || err.message.indexOf(throwMsg) >= 0;
             });
           } else {
-            runTask();
+            if (task.exec.code) {
+              let result = run(task);
+              if (result && typeof result.then === "function") {
+                return result;
```

Comment:
Basically just returning the result of the `runExec(execOpts, execCode);` if it's a promise. We could also just always return